File: task-managment/spring-web-proektna/src/main/java/com/example/springwebproektna/model/Task.java
package com.example.springwebproektna.model;

import com.fasterxml.jackson.annotation.JsonFormat;
import org.springframework.data.annotation.Id;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDate;
import java.util.Date;

public class Task {
    @Id
    private String id;
    private String taskName;
    private String taskDescription;
    private String channelId;
    private String progressBar;
    private String dueDate;
    private String column;
    private int position;

    public String getTaskName() {
        return taskName;
    }

    public void setTaskName(String taskName) {
        this.taskName = taskName;
    }

    public String getTaskDescription() {
        return taskDescription;
    }

    public void setTaskDescription(String taskDescription) {
        this.taskDescription = taskDescription;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getChannelId() {
        return channelId;
    }

    public void setChannelId(String channelId) {
        this.channelId = channelId;
    }

    public String getProgressBar() {
        return progressBar;
    }

    public void setProgressBar(String progressBar) {
        this.progressBar = progressBar;
    }

    public String getDueDate() {
        return dueDate;
    }

    public void setDueDate(String dueDate) {
        this.dueDate = dueDate;
    }

    public String getColumn() {
        return column;
    }

    public void setColumn(String column) {
        this.column = column;
    }

    public int getPosition() {
        return position;
    }

    public void setPosition(int position) {
        this.position = position;
    }
}

File: task-managment/web-proektna/src/app/model/Task.ts
export class Task {
  id: string;
  taskName: string;
  taskDescription: string;
  channelId: string;
  progressBar: string;
  dueDate: string;
  column: string;
  position: number;

  constructor(id: string, taskName: string, taskDescription: string, channelId: string, progressBar: string, dueDate: string, column: string, position: number) {
    this.id = id;
    this.taskName = taskName;
    this.taskDescription = taskDescription;
    this.channelId = channelId;
    this.progressBar = progressBar;
    this.dueDate = dueDate;
    this.column = column;
    this.position = position;
  }
}
